jupyter/minimal/ubi8-python-3.8/requirements.in

```diff
@@ -1 +1,60 @@
+# Minimal notebook requirements
 jupyterlab==3.2.4
+jupyter_server==1.4.1
+notebook==6.4.12
+setuptools==65.4.0
+pip==22.2.2
+
+# JupyterLab extensions
+# A JupyterLab extension for version control using git
+jupyterlab-git==0.39.2
+# Jupyter Kernel Gateway is a web server that supports different mechanisms for spawning and communicating with Jupyter kernels.
+jupyter_kernel_gateway==2.4.0
+# JupyterLab extension for dependency management and optimization created to guarantee reproducibility of Jupyter notebooks.
+jupyterlab-requirements>=0.6.4
+# This extension provides control over the notebook dependencies.The main goals of the project are the following: 1.manage notebook requirements without leaving the notebook 2.provide a unique and optimized* environment for each notebook
+jupyter-nbrequirements==0.7.3
+# The jupyter_nbextensions_configurator jupyter server extension provides graphical user interfaces for configuring which nbextensions are enabled (load automatically for every notebook), and display their readme files.
+jupyter_nbextensions_configurator==0.5.0
```

Comment:
[This extension doesn't work with JupyterLab](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html#more-complex-setups):

```
Most nbextensions here should work fine with jupyterhub (because jupyterhub spawns regular notebook servers for each individual user), but won’t work with jupyterlab (because the jupyterlab javascript framework is different to notebook’s, and still rapidly changing under active development). 
```

We can install extensions in JupyterLab out of the box:

![image](https://user-images.githubusercontent.com/27858594/193019547-0541ed6a-fe3c-4f2c-9c2b-628442e27e73.png)